src/Components/Checkout.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+
+export default class Checkout extends Component {
+    constructor(){
+        super()
+        this.state ={
+            firstName: '',
+            lastName: '',
+            email: '',
+            creditCard: '',
+            zipCode: ''
+        }
+    }
+    checkoutForm = (event) => {
+        event.preventDefault()
+        const {firstName, lastName, email, creditCard, zipCode} = this.state
+        if (!firstName || !lastName || !email || !creditCard || !zipCode) {
+            window.alert("Input is not valid")
+        } else if (creditCard.length !== 16){
+            window.alert("Credit card number is not valid")
+        } else if (zipCode.length !== 5){
+            window.alert("Zip code is not valid")
+        } else {
+            //window alert does not follow jsx rules ${}
+            window.alert(`Purchase complete! \n You will be charged $${this.props.realTotal.toFixed(2)}.`)
+        }
+    }
+    handleChange = (e) => {        
+        this.setState({ [e.target.id] : e.target.value})
```

Comment:
using ID here is good and totally works, but the safer way to do it would be to use `.name`. It also requires you to have a `name=""` attribute on the each input.